client/cypress/plugins/index.js

```diff
@@ -1 +1,5 @@
-module.exports = () => {};
+const percyHealthCheck = require('@percy/cypress/task'); // eslint-disable-line import/no-extraneous-dependencies, import/no-unresolved
```

Comment:
What's this?

Reply:
I forgot to mention, but percy-cypress lib from version 2 is requiring this. It's the Step 2 in their [guide](https://docs.percy.io/docs/cypress#section-setup).